optimum/onnxruntime/optimization.py

```diff
@@ -89,7 +89,7 @@ def from_pretrained(
         config = None
         if isinstance(model_or_path, ORTModel):
             from_ortmodel = True
-            if isinstance(model_or_path, ORTModelForSeq2SeqLM):
+            if isinstance(model_or_path, (ORTModelForSeq2SeqLM, ORTModelForSpeechSeq2Seq)):
```

Comment:
It shall be the same situation for `ORTModelForVision2Seq` isn't it? Why not just use the `ORTModelForConditionalGeneration` base case to check if it is a seq2seq case?

Reply:
Yes that makes sense! I forgot they share a common base. We should probably also test some vision models and add them to [`ORTConfigManager._conf`](https://github.com/huggingface/optimum/blob/369f44edd38670c68414aa954b9fcd493fa27d7e/optimum/onnxruntime/utils.py#L97).